src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.cs

```diff
@@ -201,5 +201,32 @@ internal void AppendArgumentsTo(ref ValueStringBuilder stringBuilder)
                 stringBuilder.Append(Arguments);
             }
         }
+
+        internal sealed class NullNotAllowedCollection<TCollectionItem> : Collection<TCollectionItem> where TCollectionItem : class
```

Comment:
This is a copy from https://github.com/dotnet/runtime/blob/main/src/libraries/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/NullNotAllowedCollection.cs.

Where can I put a file with this class to use it in both places?

Reply:
Perhaps in `src/libraries/Common/src/System/Collections/Generic/`? with the following lines in csproj (after line 36):
```csproj
    <Compile Include="$(CommonPath)System\Collections\Generic\NullNotAllowedCollection.cs"
             Link="Common\System\Collections\Generic\NullNotAllowedCollection.cs" />
```

Reply:
Thanks for the suggestion. I've moved it in the `Common` folder.